jersey/src/main/java/com/netflix/conductor/server/resources/MetadataResource.java

```diff
@@ -29,12 +29,16 @@
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 
+import com.google.common.base.Preconditions;
 import com.netflix.conductor.common.metadata.tasks.TaskDef;
 import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
 import com.netflix.conductor.service.MetadataService;
 
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
+import org.apache.commons.lang3.StringUtils;
+
+import static java.lang.Thread.*;
```

Comment:
Seems like the new imports are not being used? Also, please try to avoid importing with `*`